designs/umbra/src/base.mjs

```diff
@@ -0,0 +1,814 @@
+    .angle(paths.simpleBackCurve.shiftFractionAlong(curvePointFraction + epsilon))
+
+  /*
+   * Determine control point for maximum coverage
+   */
+  points.backCurveCenterMax = new Point(
+    points.sideLegBase.x * 0.7,
+    points.cfMiddle.shiftFractionTowards(points.sideLegBaseBack, 0.3).y
+  )
+  /*
+   * Determine control point for minimum coverage (thong-like)
+   */
+  points.backCurveCenterMin = points.cfWaistbandBack
+    .shiftFractionTowards(points.cfMiddle, 0.5)
+    .shift(0, minFabricWidth / 2)
+
+  /* This is the additional point */
+  points.sideFullnessBack = interpolatePoint(
+    points.backCurveCenterMax,
+    points.backCurveCenterDefault,
+    points.backCurveCenterMin,
+    options.backExposure
+  )
+
+  if (points.sideFullnessBack.y > backMaxY) {
+    points.sideFullnessBack = new Point(points.sideFullnessBack.x, backMaxY)
+  }
+
+  /* Determine the angle for the line at that point */
+  const shiftAngle = interpolateValue(160, curvePointAngle, 90, options.backExposure)
+  const shiftAmount =
+    points.cfSeat.dist(points.sideSeat) * interpolateValue(0.5, 0.1, 0.5, options.backExposure)
+
+  /* Determine control points */
+  points.sideLegCp2Back = points.sideFullnessBack.shift(shiftAngle, -shiftAmount)
+  if (points.sideLegCp2Back.y > backMaxY) {
+    points.sideLegCp2Back = new Point(points.sideLegCp2Back.x, backMaxY)
+  }
+  points.gussetBackCp1 = points.sideFullnessBack.shift(shiftAngle, shiftAmount)
+  points.gussetBackCp2 = points.sideGusset.shift(
+    90,
+    points.sideFullnessBack.dy(points.sideGusset) / 4
+  )
+  points.gussetFrontCp = points.sideGusset.shift(
+    90,
+    points.sideLegCpFront.dy(points.sideGusset) * options.frontExposure
+  )
+
+  /*
+   * Construct leg curve for the back part
+   */
+  paths.backCurve = new Path()
+    .move(points.sideLegBack)
+    .curve(points.sideLegCp1Back, points.sideLegCp2Back, points.sideFullnessBack)
+    .curve(points.gussetBackCp1, points.gussetBackCp2, points.sideGusset)
+    .hide()
+
+  /*
+   * Determine split point for crotch seam
+   */
+  const intersectsY = paths.backCurve.intersectsY(points.cfBackGusset.y)[0]
+
+  let backCurveParts = []
+  if (intersectsY && !Array.isArray(intersectsY)) {
+    points.backGussetSplit = intersectsY
+    backCurveParts = paths.backCurve.split(points.backGussetSplit)
+  }
+  if (!(backCurveParts && backCurveParts.length > 1)) {
+    points.backGussetSplit = points.sideGusset
+    backCurveParts = [paths.backCurve]
+  }
+
+  /*
+   * This is the part of the back curve that will be used for the back part
+   */
+  paths.elasticLegBack = backCurveParts[0].hide()
+
+  paths.back = new Path()
+    .move(points.cfWaistbandDipBack)
+    .curve(points.cfWaistbandDipCp2Back, points.cfWaistbandDipCp1Back, points.sideWaistbandBack)
+    .join(paths.elasticLegBack)
+    .line(points.cfBackGusset)
+    .hide()
+
+  // Determine center position of bulge
+  // TODO: Add measurement or make adjustable?
```

Comment:
This is a minor open TODO, I don't think it should be blocking the merge, and I don't see a problem addressing it later,  just wanted to make you aware of it.